FluentTerminal.SystemTray/Services/ConPty/ConPtySession.cs

```diff
@@ -80,7 +83,8 @@ private void ListenToStdOut()
                 {
                     do
                     {
-                        var buffer = new byte[1024];
+                        int bufferSize = 1024 > _terminalSize.Columns * _terminalSize.Rows * 4 ? 1024 : _terminalSize.Columns * _terminalSize.Rows * 4;
```

Comment:
I would suggest Math.Max to make this more readable

Reply:
@felixse sounds great, thank you! Have updated it